src/schema/comments.ts

```diff
@@ -659,9 +739,13 @@ export const resolvers: IResolvers<any, Context> = {
         if (comment.userId !== ctx.userId) {
           throw new ForbiddenError("Cannot edit someone else's comment");
         }
+        const post = await comment.post;
+        const source = await post.source;
```

Comment:
```suggestion
        const [post, source] = await Promise.all([comment.post, comment.source]);
```

Reply:
Oh, wow. Didn't know this one. Amazing!

Reply:
I tried this but it failed since the comment doesn't have the property of `source` - we will have to make it a sequence.